### PR TITLE
refactor(multi_object_tracker): apply clang-tidy check

### PR DESCRIPTION
**Pull request by soblin**
## Description

https://github.com/orgs/autowarefoundation/discussions/4876

## Related links

https://github.com/orgs/autowarefoundation/discussions/4876

## Tests performed

build pass

## Interface changes

none

## Effects on system behavior

none

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [X] The PR follows the [pull request guidelines].
- [X] The PR has been properly tested.
- [X] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [X] There are no open discussions or they are tracked via tickets.
- [X] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by YoshiRi** [comment]
@soblin Does this PR wants to fix something like: `constexpr variable 'hoge' must be initialized by a constant expression` ?

Then I think some parts of the change are not necessary, right?

**Review by YoshiRi — APPROVED** [review]
LGTM because it should works.

**Review by technolojin — APPROVED** [review]
LGTM!

I was [worked on it](https://github.com/autowarefoundation/autoware.universe/pull/7271), but it became too large refactoring. It is OK to be merged first.

File: perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp [hunk 1]
```diff
@@ -123,13 +123,12 @@ BicycleTracker::BicycleTracker(
...
     if (!object.kinematics.has_position_covariance) {
       // initial state covariance
-      constexpr double p0_stddev_x = 0.8;  // in object coordinate [m]
-      constexpr double p0_stddev_y = 0.5;  // in object coordinate [m]
-      constexpr double p0_stddev_yaw =
-        tier4_autoware_utils::deg2rad(25);  // in map coordinate [rad]
-      constexpr double p0_cov_x = p0_stddev_x * p0_stddev_x;
-      constexpr double p0_cov_y = p0_stddev_y * p0_stddev_y;
-      constexpr double p0_cov_yaw = p0_stddev_yaw * p0_stddev_yaw;
+      const double p0_stddev_x = 0.8;                                  // in object coordinate [m]
```
Comment (YoshiRi) [inline perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp:126]: [imo]
This part should no problem

**Pull request closed without merging**